src/main/java/org/folio/dew/batch/bursarfeesfines/service/impl/BursarExportServiceImpl.java

```diff
@@ -72,6 +86,19 @@ public void transferAccounts(List<Account> accounts, BursarFeeFines bursarFeeFin
       log.error("Can not create query for batch job, cause outStandingDays are null.");
       return Collections.emptyList();
     }
+
+    if (users.size() < bucketSize) {
```

Comment:
change from `<` to `<=`, otherwise the `findAccountsLessThanBucketSizeTest` will slip through

Reply:
updated